config/initializers/devise.rb

```diff
@@ -41,7 +41,7 @@
   # if you set :request_keys to [:subdomain], :subdomain will be used on authentication.
   # The same considerations mentioned for authentication_keys also apply to request_keys.
   # config.request_keys = []
-
+  config.omniauth :facebook, "1187829091254806", "33c993c6ce557656d32b76c7ca53fa9e"
```

Comment:
You don't need to add the app id and secret as it is already added in this line https://github.com/NIT-dgp/cat-forum/blob/github_master/config/initializers/devise.rb#L262 in the form of secret tokens. You just need to add them in the `app_id` and `app_secret` fields in secrets.yml